DOCS/man/osc.rst

```diff
@@ -293,7 +293,7 @@ Configurable Options
 
     String that supports property expansion that will be displayed as
     OSC title.
-    ASS tags are escaped, and newlines and trailing slashes are stripped.
+    ASS tags are escaped and newlines are converted to spaces.
```

Comment:
This is not a behaviour change but just bringing the documentation up-to-date, right?

Reply:
It is a behavior change as the escaping is updated to be complete as described in the second commit message, though the documentation was also incorrect in saying that newlines are stripped. Actual newlines are now converted to spaces instead of "\\n", which is escaped instead, and backslashes are escaped instead of just stripping trailing ones.